scripts/ci/install-minikube.sh

```diff
@@ -43,9 +43,37 @@ check_minikube() {
         fi
 
         sudo systemctl start docker
+:x
+
+downgrade_iptables() {
+        # HACK HACK HACK: kube-proxy passes iptables-restore flag -w5 which is
+        # ignored in v1.6.1 (FC27) but failed v1.6.2 (FC28), thus we downgrade
+        # iptables on FC28. Once this issue is fixed in k8s is hack witll be
+        # removed.  issue https://github.com/NixOS/nixpkgs/issues/35544, it's
+        # patch has not yet been released in default kubernetes server used by
+        # minikube.
+        . /etc/os-release
+        if [ "$ID" == "fedora" -a "$VERSION_ID" == "28" ]; then
+                sudo iptables-restore -h
+                IPT_VER=1.6.1
+                IPT_PKG=iptables-$IPT_VER
+                IPT_TGZ=$IPT_PKG.tar.bz2
+                IPT_URL=http://www.netfilter.org/projects/iptables/files/$IPT_TGZ
+                rm -rf $IPT_TGZ $IPT_PKG
+                wget $IPT_URL
+                tar xvfj $IPT_TGZ
+                cd $IPT_PKG
+                sudo dnf install -y libnfnetlink-devel libmnl-devel libnftnl-devel
+                ./configure
+                make
+                sudo make install
```

Comment:
Lucky the CI are not in fc28 yet : that's the kind of hack that will break the CI integrity ....

futhermore you didn't check if iptables has been already downgraded, 2/ why not downgrading the package (much cleaner, and rpm dependency will be checked if it's possible)

In all case would be better to write a wrapper that (fix the bad k8s parameters) for iptables-restore instead of downgrading a version .... could you reply "did downgrading iptables will/would break something else ?"

Reply:
only "-w5" => "-w 5" parametter is impacted (fixed) by k8s 
https://github.com/kubernetes/kubernetes/commit/34ce573e9992ecdbc06dff1b4e3d0e9baa8353dd.patch

Reply:
ok, implemented a wrapper to iptables-restore which eliminates options of form -w?*; when fix is part of k8s server we will remove the hack.